### DISCUSSION TITLE
MacOS 13

### DISCUSSION THREAD

**Discussion by karosc** [MarkusJx/install-boost#39] [Q&A]
Is there any plan to add MacOS 13 to the [supported platform_versions](https://github.com/MarkusJx/prebuilt-boost/blob/805c68e361f8944bd40b4372164171db9e4d2275/versions-manifest.json#L1028-L1053)? I guess it is [still in beta](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources), but just wondering what the plan is.

**Comment by MarkusJx** [comment]
I've also noticed this as I've removed the now defunct version 10.15 from my build configuration. Anyways, macos 13 builds are already present with [boost version 1.82.0](https://github.com/MarkusJx/prebuilt-boost/releases/tag/1.82.0) and will be supported with all new boost versions I choose to build (which happens every time someone reminds me to build a specific version). The new versions supporting macOS 13 will be available in a few hours once the build for boost 1.83.0 succeeds.

If you need a specific version of boost compiled with support for macOS 13, just let me know.
